src/components/Articles.jsx

```diff
@@ -1,9 +1,28 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { getArticles } from '../actions/articles';
 
 const Articles = () => {
+	const [articles, setArticles] = useState([]);
+
+	useEffect(() => {
+		getArticles()
+			.then((data) => {
+				console.log(data.articles);
+				setArticles([...data.articles]);
+			})
+			.catch((err) => {
+				console.log(err);
+			});
+	}, []);
+
 	return (
-		<div>
+		<div className='h-screen'>
 			<h1>Articles</h1>
+
+			{articles &&
+				articles.map((article) => {
+					return <p>Article: {article.article_id}</p>;
```

Comment:
would show more than just article ID here

Reply:
I do plan on showing more. I just wanted to ensure was on correct path before moving forward. As for the hamburger menu, I am not sure as locally it shows the menu when you click it. I am confused as to why it doesn;t for you.